### DISCUSSION TITLE
Gatsby 잘 사용하기 (지금까지 내가 Gatsby를 사용하며 겪은 삽질과 꿀팁들, Gatsby로 블로그 만들기, React) - 정현수 기술 블로그

### DISCUSSION THREAD

**Discussion by giscus** [junghyeonsu/junghyeonsu.dev.comment#40] [Announcements]
# Gatsby 잘 사용하기 (지금까지 내가 Gatsby를 사용하며 겪은 삽질과 꿀팁들, Gatsby로 블로그 만들기, React) - 정현수 기술 블로그

Gatsby를 사용하며 겪은 삽질과 꿀팁들을 공유합니다.

https://junghyeonsu.com/posts/gatsby-shovels-and-hacks/

**Comment by areumsheep** [comment]
저도 게츠비로 블로그를 개발하고 있는데 유용한 정보들이 많네요! 저도 적용해봐야겠어요!!
다음 삽질기도 기대할게요...! 🥹

**Comment by deli-ght** [comment]
와 글 되게 유익하네요! 노션에 이슈 정리하는 법도 따라해 봐야겠어요! 좋은 글 감사해요 :)

**Comment by moonkorea00** [comment]
shovels and hacks 👍

**Comment by jungsikjeong** [comment]
저 개인적으로 궁금한게 
오른쪽 상단 어바웃누르면 새로운 웹 사이트로 소개하셨던데
그거는 직접 만드신걸까요?

**Reply by junghyeonsu** [reply]
넵 블로그랑은 별개의 도메인을 가지고 있고 따로 개발됐어요. 단순히 a 링크로 연결시켜놓은거에요ㅎㅎ

**Comment by HodooSeungMo** [comment]
올해 목표가 블로그 개발이라 gatsby로 작성한 준 블로그로 구경왔어요. 
역시 👍

**Comment by HodooSeungMo** [comment]
올해 목표가 블로그 개발이라 gatsby로 작성한 준 블로그로 구경왔어요.
역시 👍

**Comment by sungpaks** [comment]
내용도 유익하고 정리도 깔끔하고 글이 너무 잘 읽힙니다.. 
저도 Gatsby로 블로그 운영하며 만지작하고 있는데 도움 많이 얻어갑니다!

**Reply by junghyeonsu** [reply]
읽어주셔서 감사합니다!!!
